src/app/home/home-banner/home-banner.component.html

```diff
@@ -13,9 +13,7 @@ <h5 class="banner-description">{{info.subtitle}}</h5>
     </div>
   </div>
   <div class="first-image-banner">
-      <div [ngSwitch]="showMedia">
-          <img class="image" *ngSwitchCase="this.IMAGE_TYPE" width="100%" [src]="info.image"/>
-          <img class="video" *ngSwitchCase="this.VIDEO_TYPE" width="100%" [src]="info.video"/>
-      </div> 
+      <img class="banner-image" [ngClass]="info.class" width="100%" [src]="info.image"/>
```

Comment:
Any particular reason to switch for CSS hiding the component, instead of using the ngSwitch as it was initially?

Reply:
Showing / hiding the elements using `resize` with javascript is expensive. CSS handles this better.